platform/darwin/src/MGLPolyline.mm

```diff
@@ -73,38 +73,34 @@ - (CLLocationCoordinate2D)coordinate {
                     return coordinates[i];
                 }
                 to = MGLRadianCoordinateFromLocationCoordinate(coordinates[i - 1]);
-                MGLRadianDirection direction = [self direction:from to:to] - 180;
+                CLLocationDirection direction = [self direction:from to:to] - 180;
                 MGLRadianCoordinate2D otherCoordinate = MGLRadianCoordinateAtDistanceFacingDirection(from,
-                                                                                                     overshoot/MGLMetersPerRadian,
+                                                                                                     overshoot/mbgl::util::M2PI * mbgl::util::EARTH_RADIUS_M,
```

Comment:
> coordinate(at:facing:) divides distance by metersPerRadian. That’s very different than dividing overshoot by 2π and multiplying the result by mbgl::util::EARTH_RADIUS_M.

Right that's why I said it was a paste mistake, it should be this way:
```
MGLRadianCoordinate2D otherCoordinate = MGLRadianCoordinateAtDistanceFacingDirection(from,
                                                                                                    overshoot/MGLMetersPerRadian,
                                                                                                     MGLRadiansFromDegrees(direction));
```

I mistakenly replaced a constant I had before called:
`const double MGLMetersPerRadian = 6373000.0;`

This is the underlaying impl I didn't consider I needed another function since this is the only place we are using this code.
```
/// Returns a coordinate a certain Haversine distance away in the given direction.
    func coordinate(at distance: CLLocationDistance, facing direction: CLLocationDirection) -> CLLocationCoordinate2D {
        let radianCoordinate = RadianCoordinate2D(self).coordinate(at: distance / metersPerRadian, facing: direction.toRadians())
        return CLLocationCoordinate2D(radianCoordinate)
    }
```